.github/commitlint.config.js

```diff
@@ -0,0 +1,37 @@
+const maxTypeNumber = 3
+
+const validateTypeNums = (parsedCommit) => {
+    if (!parsedCommit.type) {
+        return [false, 'invalid commit message']
+    }
+
+    return [
+        parsedCommit.type.split(' ').length <= maxTypeNumber,
+      `type must not be more than ${maxTypeNumber}`,
```

Comment:
`maxTypeNumber` is not mandatory right now.

Reply:
removed